analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -56,8 +56,11 @@ public class SymbolicExecutionRunner : SymbolicExecutionRunnerBase
             new InitializationVectorShouldBeRandom(),
             new HashesShouldHaveUnpredictableSalt());
 
-        protected override ImmutableDictionary<DiagnosticDescriptor, RuleFactory> AllRules => ImmutableDictionary<DiagnosticDescriptor, RuleFactory>.Empty
-            .Add(LocksReleasedAllPaths.S2222, CreateFactory<LocksReleasedAllPaths>());
+        protected override ImmutableDictionary<DiagnosticDescriptor, RuleFactory> AllRules => new Dictionary<DiagnosticDescriptor, RuleFactory>
+        {
+            { RuleChecks.LocksReleasedAllPaths.S2222, CreateFactory<RuleChecks.LocksReleasedAllPaths>() },
+            { RuleChecks.NullPointerDereference.S2259, CreateFactory<RuleChecks.NullPointerDereference>() },
+        }.ToImmutableDictionary();
```

Comment:
It's easier maintainable with the trailing comma this way (copy line) and should be slightly better performing allocation-wise.